course_content/notebooks/matplotlib_intro.ipynb

```diff
@@ -396,18 +441,20 @@
        "text": [
-        "<matplotlib.figure.Figure at 0x4911550>"
+        "<matplotlib.figure.Figure at 0x3ba1dd0>"
        ]
       }
      ],
-     "prompt_number": 10
+     "prompt_number": 35
     },
     {
      "cell_type": "code",
      "collapsed": false,
      "input": [
+      "# Pcolor is a pseudocolour plot whose cell colours are determined by the values of the input array.\n",
+      "# Pcolormesh is a faster mehcanism for constructing a pcolor.\n",
```

Comment:
Again.

Typo: "mehcanism" --> "mechanism".

I also don't think this line tells all the story it needs to - `pcolormesh` is also a superior algorithm, although it can introduce some very weird effects depending on how you set the interpolation and other options.